.github/workflows/gradle.yml

```diff
@@ -25,49 +25,60 @@ jobs:
   build:
     name: Build for JDK ${{ matrix.java }}
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name != 'Discord4J/Discord4J' }}
+    if: ${{ github.event_name == 'push' || (github.event.pull_request.head.repo.full_name != 'Discord4J/Discord4J' && !contains(toJson(github.event.pull_request.labels.*.name), 'PR depends on PR')) }}
```

Comment:
Not looking to include this change now, it's okay to have the red X indicating pending work

Reply:
hmmm and maybe move this to another step and if has the label cancel the work (still show X)
mostly this was because when making PR with dependency PR for any commit i get a email (maybe need disable this) notici the failed build.. and yeah failed because need another things for work.

Reply:
Edited, now if the event has the label for depend of pr then the step mark the workflow to failed avoiding the build thing but setting a failed message like this.
![image](https://user-images.githubusercontent.com/3602279/230732709-a898dafb-82fd-4e61-b2f5-1b2765e3f814.png)

Reply:
It's great to know we can have that kind of message but I guess you would still get the email if it errors like this

For now, remove the changes about detecting the label at all. I'd like to see a compilation error pointing out what's wrong or missing in code, instead of defensively performing this validation.

Reply:
Okay i remove this.. but one purpose of this PR was avoid get many email for failed for every commit to a PR who know its supposed to fail because still need another PR, but can test the previous workarround using ci-skip for every commit....

Reply:
![image](https://user-images.githubusercontent.com/3602279/230783210-e85fa51f-4911-47d6-bc29-3305b4e4149b.png)